unit-tests/native/src/test/scala/scala/scalanative/IssuesTest.scala

```diff
@@ -580,6 +580,16 @@ class IssuesTest {
     free(null)
   }
 
+  @Test def `can initialize lazy vals using linktime if`() = {
```

Comment:
Could this just be toggle case? I ask because I spent a bunch of time on Scala.js normalizing all the tests. The only exception is `_Issue341` as a postfix. See https://github.com/scala-js/scala-js/pull/4266